README.md

```diff
@@ -5,7 +5,8 @@
 <div align="right">
   <sup>
     <a href="#the-challenge">English</a> ·
-    <a href="#お題">日本語</a>
+    <a href="#お題">日本語</a> ·
+    <a href="#tantangan">Indonesia</a> ·
```

Comment:
Maybe the last `·` should be excluded, as in the original version of the README.md it is only used as a delimiter _between_ the different languages.